app/assets/javascripts/app/app_router.coffee

```diff
@@ -0,0 +1,21 @@
+Router = Backbone.Router.extend {
+  routes:
+    ':name'   : 'show'
+
+  show: (name) ->
+    @showPage(name)
+
+  showPage: (name) ->
+    window.peopleList = new PeopleList
+    peopleList.fetch().then ->
+      $('.card').remove()
+      $.each(peopleList._byId, (_prop, person_data) ->
```

Comment:
Another option here is to use native browser array iterator fns such as

`[].forEach(...)`

A lot of options :)